storage/innobase/xtrabackup/doc/source/installation/compiling_xtrabackup.rst

```diff
@@ -31,37 +31,62 @@ In Debian-based distributions, you need to:
 
 .. code-block:: bash
 
- $ apt-get install build-essential flex bison automake autoconf \
-    libtool cmake libaio-dev mysql-client libncurses-dev zlib1g-dev \
-    libgcrypt11-dev libev-dev libcurl4-gnutls-dev vim-common
+   $ apt install build-essential flex bison automake autoconf \
+   libtool cmake libaio-dev mysql-client libncurses-dev zlib1g-dev \
+   libgcrypt11-dev libev-dev libcurl4-gnutls-dev vim-common
 
-
-In ``RPM``-based distributions, you need to:
+In ``RPM``-based distributions, you need to install and enable `the
```

Comment:
Since dev toolset 7 is needed on rhel 6 and 7, but is not needed in fedora and rhel 8, can we just ask to check their GCC version and if it is older than 5.3, then install dev toolset?